# Admittance parameters for two mass damper systems
# (the arm and the platform) with params (mass_arm, damping_arm)
# and (mass_platform, damping_platform) respectively, connected by a
# spring-damper with parameters (damping_coupling, stiffness_coupling,
# equilibrium_point_spring)
mass_platform: [1,0,0,0,0,0,
                0,1,0,0,0,0,
                0,0,1,0,0,0,
                0,0,0,1,0,0,
                0,0,0,0,1,0,
                0,0,0,0,0,1]
mass_arm: [2,0,0,0,0,0,
           0,2,0,0,0,0,
           0,0,2,0,0,0,
           0,0,0,2,0,0,
           0,0,0,0,2,0,
           0,0,0,0,0,2]
damping_coupling: [0,0,0,0,0,0,
                   0,0,0,0,0,0,
                   0,0,0,0,0,0,
                   0,0,0,0,0,0,
                   0,0,0,0,0,0,
                   0,0,0,0,0,0]
damping_platform: [5,0,0,0,0,0,
                   0,5,0,0,0,0,
                   0,0,5,0,0,0,
                   0,0,0,0,0,0,
                   0,0,0,0,0,0,
                   0,0,0,0,0,5]
damping_arm: [12,0,0,0,0,0,
              0,12,0,0,0,0,
              0,0,12,0,0,0,
              0,0,0,10,0,0,
              0,0,0,0,10,0,
              0,0,0,0,0,10]
stiffness_coupling: [10,0,0,0,0,0,
                     0,20,0,0,0,0,
                     0,0,10,0,0,0,
                     0,0,0,10,0,0,
                     0,0,0,0,10,0,
                     0,0,0,0,0,10]
# [position (x, y, z)  orientation (x, y, z, w)]
equilibrium_point_spring: [0.1,0.4,0.5,-0.593599, -0.566706, -0.369419, 0.435908]

